WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.m

```diff
@@ -1257,13 +1257,19 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
 
 - (void)handleNotificationChange:(NSNotification *)notification
 {
-    NSSet *updated = notification.userInfo[NSUpdatedObjectsKey];
-    NSSet *refreshed = notification.userInfo[NSRefreshedObjectsKey];
+    NSSet *updated      = notification.userInfo[NSUpdatedObjectsKey];
+    NSSet *refreshed    = notification.userInfo[NSRefreshedObjectsKey];
+    NSSet *deleted      = notification.userInfo[NSDeletedObjectsKey];
```

Comment:
Does this set contain `Notification` objects? That seems weird it would have NSManagedObjects in the user info payload but I could be misinterpreting it.